help_to_heat/frontdoor/schemas.py

```diff
@@ -1,288 +1,380 @@
+#     "loft": {"prev": "wall-insulation", "next": "loft-access"},
+#     "loft-access": {"prev": "loft", "next": "loft-insulation"},
+#     "loft-insulation": {"prev": "loft-access", "next": "summary"},
+#     "epc-ineligible": {"prev": "epc", "next": None},
+#     "ineligible": {"prev": "household-income", "next": None},
+#     "northern-ireland": {"prev": "country", "next": None},
+#     "bulb-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "utility-warehouse-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "shell-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "applications-closed": {"prev": "supplier", "next": None},
+#     "benefits": {"prev": "council-tax-band", "next": "household-income"},
+#     "park-home": {"prev": "own-property", "next": "address"},
+#     "park-home-main-residence": {"prev": "park-home", "next": "address"},
+#     "park-home-application-closed": {"prev": "park-home-main-residence", "next": None},
+#     "referral-already-submitted": {"prev": "address", "next": "council-tax-band"},
+# }
+#
+# page_prev_next_map_park_home = {
+#     "address-select": {"prev": "address", "next": "epc"},
+#     "epc-select": {"prev": "address", "next": "epc"},
+#     "address-manual": {"prev": "address", "next": "epc"},
+#     "northern-ireland": {"prev": "country", "next": None},
+#     "bulb-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "utility-warehouse-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "shell-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "applications-closed": {"prev": "supplier", "next": None},
+#     "application-closed-utility-warehouse": {"prev": "supplier", "next": None},
+#     "park-home-application-closed": {"prev": "park-home-main-residence", "next": None},
+#     "epc-ineligible": {"prev": "epc", "next": None},
+#     "ineligible": {"prev": "household-income", "next": None},
+#     "referral-already-submitted": {"prev": "address", "next": "epc"},
+# }
+#
+# page_prev_next_map_social_housing = {
+#     "address-select": {"prev": "address", "next": "epc"},
+#     "epc-select": {"prev": "address", "next": "epc"},
+#     "address-manual": {"prev": "address", "next": "epc"},
+#     "loft": {"prev": "wall-insulation", "next": "loft-access"},
+#     "loft-access": {"prev": "loft", "next": "loft-insulation"},
+#     "loft-insulation": {"prev": "loft-access", "next": "summary"},
+#     "epc-ineligible": {"prev": "epc", "next": None},
+#     "northern-ireland": {"prev": "country", "next": None},
+#     "bulb-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "utility-warehouse-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "shell-warning-page": {"prev": "supplier", "next": "own-property"},
+#     "applications-closed": {"prev": "supplier", "next": None},
+#     "referral-already-submitted": {"prev": "address", "next": "epc"},
+# }
```

Comment:
Why has this commented chunk been left in?

Reply:
this will be resolved as part of [PC-1232](https://beisdigital.atlassian.net/browse/PC-1232), leaving in for reference as some of this information will need to be kept